jobs/e2e_node/containerd/image-config-v2.yaml

```diff
@@ -0,0 +1,10 @@
+images:
+  ubuntu:
+    image_family: pipeline-1-29
+    project: ubuntu-os-gke-cloud
+    metadata: "user-data</home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/containerd/init.yaml,cni-template</home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/containerd/cni.template,containerd-config</home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/containerd/config.toml,registry-config-docker</home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/containerd/hosts-docker.toml"
```

Comment:
containerd/config.toml -> containerd/config-v2.toml right?

Reply:
Thanks for catching it @dims , Updated!